projects/core/src/user/store/reducers/user-addresses.reducer.ts

```diff
@@ -18,22 +17,19 @@ export function reducer(
       return {
         ...state,
         list,
-        isLoading: false,
         isActionProcessing: false
       };
     }
 
     case fromUserAddressesAction.LOAD_USER_ADDRESSES_FAIL: {
       return {
-        ...state,
-        isLoading: false
+        ...state
       };
     }
 
     case fromUserAddressesAction.LOAD_USER_ADDRESSES: {
```

Comment:
this `case` can be removed

Reply:
In order to have this case covered, I didn't remove it.